.github/workflows/ci.yml

```diff
@@ -11,6 +11,7 @@ on:
 env:
  BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
  EXCLUDE_ENTERPRISE: true
+ GO_VERSION: 1.24.0
```

Comment:
The project itself builds on go 1.22 as designated in [go.mod file](https://github.com/mattermost/mattermost-plugin-boards/blob/main/go.mod). Would this cause a problem and should we upgrade the golang version required by the plugin?

Reply:
1.22 is deprecated. We should definitely move out of it.